handlers/gnatsd.go

```diff
@@ -52,6 +52,9 @@ func (lc *GnatsdHandler) Handle(event *types.Event, conf *types.HandlerConfig) (
 
 	fmt.Printf("[gnatsd] phase=%s topic=%s %+v\n", event.Type, topic, event.Data)
 
+	// Flushes the connection ensuring the last event gets published before the app terminates
```

Comment:
I would like GnatsHandler to expose a Close() method, so allow the caller to control the life cycle of instance